editor-components/modal/formmodal1/formmodal1.tsx

```diff
@@ -0,0 +1,353 @@
+                      displayer: "Input",
+                      value: [
+                        {
+                          type: "string",
+                          key: "placeholder",
+                          displayer: "Placeholder",
+                          value: "+1(555) 000-0000",
+                        },
+                        {
+                          type: "select",
+                          key: "type",
+                          displayer: "Type",
+                          value: "Text",
+                          additionalParams: {
+                            selectItems: ["Text", "E-mail", "Number"],
+                          },
+                        },
+                      ],
+                    },
+                  ],
+                },
+              ],
+            },
+            
+          ],
+        });
+        
+        this.addProp({
+          type: "string",
+          key: "button_text",
+          displayer: "Button Text",
+          value: "Get in touch",
+        });
+    }
+
+    getName(): string {
+        return "FormModal1";
+    }
+
+    render() {
+      const header = this.getPropValue("header", { as_string: true });
+      const context = this.getPropValue("context", { as_string: true });  
+      const inputItems = this.getPropValue("input-items")!;
+    
+      function getInputType(type: string): string {
+        switch (type) {
+          case "TextArea":
+            return "text-area";
+          case "Email":
+            return "e-mail";
+          case "Tel":
+            return "tel";
+          case "Number":
+            return "number";
+          default:
+            return "text";
+        }
+      }
+    
+      function getInputName(
+        indexOfLabel: number,
+        inputLabel: string,
+        indexOfInput: number
+      ): string {
+        const name = toObjectKey(`${indexOfLabel} ${inputLabel} ${indexOfInput}`);
+        return toObjectKey(name);
+      }
+    
+      function toObjectKey(str: string) {
+        if (/^\d/.test(str)) {
+          str = "_" + str;
+        }
+        str = str.replace(/[^a-zA-Z0-9_]/g, "_").toLowerCase();
+        return str;
+      }
+    
+      function getInitialValue() {
+        let value: any = {};
+        inputItems.map((inputItem: any, indexOfItem: number) => {
+          inputItem
+            .getPropValue("inputs")
+            ?.map((_: TypeUsableComponentProps, indexOfInput: number) => {
+              const key = getInputName(
+                indexOfItem,
+                inputItem.getPropValue("label"),
+                indexOfInput
+              );
+              value[key] = "";
+            });
+        });
+        return value;
+      }
+    
+      return(
+        <div className={this.decorateCSS("container")}>
+          <div className={this.decorateCSS("max-content")}>
+            <div className={this.decorateCSS("page")}>
+              <div className={this.decorateCSS("left-page")}>
```

Comment:
If there is no image uploaded, delete img's div. Make the other divs strech for available space.